backend/src/main/java/com/praksa/auction/dto/LogInDto.java

```diff
@@ -12,6 +14,21 @@ public LogInDto(String email, String password) {
     public LogInDto() {
     }
 
+    public String validateData() {
```

Comment:
Use JSR 380, follow this [link](https://www.baeldung.com/javax-validation). For email it would look like this:
```
@Email(message = "Email should be valid")
    private String email;
```

Reply:
For `password` you can use `@Min `